services/auth/README.md

```diff
@@ -6,7 +6,7 @@ The auth service manages permissions and credentials in a Taskcluster deployment
 
 No special configuration is required for development.
 
-Run `yarn workspace taskcluster-hooks test` to run the tess.
+Run `yarn workspace taskcluster-hooks test` to run the test.
```

Comment:
This should be plural -- there are more than one test :)

Reply:
Should be done now :)